engine/runtime-instrument-id-execution/src/main/java/org/enso/interpreter/instrument/IdExecutionInstrument.java

```diff
@@ -186,12 +186,11 @@ public void onReturnValue(EventContext context, VirtualFrame frame, Object resul
         FunctionCallInfo cachedCall = cache.getCall(nodeId);
         ProfilingInfo[] profilingInfo = new ProfilingInfo[] {new ExecutionTime(nanoTimeElapsed)};
 
+        System.out.println("onReturnValue " + nodeId + " value: " + result + " type: " + resultType);
         ExpressionValue expressionValue =
             new ExpressionValue(
                 nodeId, result, resultType, cachedType, call, cachedCall, profilingInfo, false);
-        if (expressionValue.isTypeChanged() || expressionValue.isFunctionCallChanged()) {
-          syncState.setExpressionUnsync(nodeId);
-        }
+        syncState.setExpressionUnsync(nodeId);
```

Comment:
FYI, @farmaazon, we now need to send information about all computed values to make sure the `Pending` state is changed back to _computed_ one.